### PR TITLE
Age of empires 3 cheat sheet

### PR DESCRIPTION
**Pull request by yasharmaster**
https://duck.co/ia/view/age_of_empires_3_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@yasharmaster Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/age_of_empires3.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    "name": "AgeOfEmpires3",
+    "description": "Full list of all Age of Empires 3 cheat codes",
+
+    "metadata": {
+        "sourceName": "Age of Empires 3 Wiki",
+        "sourceUrl" : "http://ageofempires.wikia.com/wiki/Cheat_Codes_(Age_of_Empires_III)"
+    },
+
+    "aliases": [
+        "age of empires 3", "aoe3", "aoe 3"
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/age_of_empires3.json]: Separate these into an alias each line

File: share/goodie/cheat_sheets/json/age_of_empires3.json [hunk 2]
```diff
@@ -0,0 +1,119 @@
+{
+    "id": "age_of_empires3_cheat_sheet",
+    "name": "AgeOfEmpires3",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/age_of_empires3.json]: Please could you put spaces between the words in the `name` property.

File: share/goodie/cheat_sheets/json/age_of_empires3.json [hunk 3]
```diff
@@ -0,0 +1,119 @@
+{
+    "id": "age_of_empires3_cheat_sheet",
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/age_of_empires3.json]: Please could you put a space after "empires" so it becomes `age_of_empires_3_cheat_sheet`

**Comment by tagawa** [comment]
Hi @yasharmaster. Thank you for submitting another Instant Answer. The checks seem to be failing because the ID and the name of the IA page have differences. If you change the filename to `age_of_empires_3.json` and make the other suggested changes, I think it will be OK.

**Comment by yasharmaster** [comment]
@tagawa I have fixed the id and name properties and renamed the file. Still there seems to be a problem in deploying. Please review the changes.

**Comment by ManrajGrover** [comment]
@yasharmaster Try renaming your file to `age-of-empires-3.json` . I had faced similar issue in one of my previous PRs. This should fix it. :blush:

**Comment by yasharmaster** [comment]
@ManrajGrover Thanks. The issue is now fixed :)

File: share/goodie/cheat_sheets/json/age-of-empires-3.json [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    "name": "Age Of Empires 3",
+    "description": "Full list of all Age of Empires 3 cheat codes",
+
+    "metadata": {
+        "sourceName": "Age of Empires 3 Wiki",
+        "sourceUrl" : "http://ageofempires.wikia.com/wiki/Cheat_Codes_(Age_of_Empires_III)"
+    },
+
+    "aliases": [
+        "age of empires 3",
```
Comment (ManrajGrover) [inline share/goodie/cheat_sheets/json/age-of-empires-3.json]: @yasharmaster Build still isn't [passing](https://travis-ci.org/duckduckgo/zeroclickinfo-goodies/builds/138735669#L595). Alias cannot be same as file name. Please remove it. :blush:

**Comment by ManrajGrover** [comment]
@yasharmaster Build still isn't [passing](https://travis-ci.org/duckduckgo/zeroclickinfo-goodies/builds/138735669#L595). Left a comment. Fixing it should clear all checks. :blush:

File: share/goodie/cheat_sheets/json/age-of-empires-3.json [hunk 5]
```diff
@@ -0,0 +1,120 @@
+{
+    "id": "age_of_empires_3_cheat_sheet",
+    "name": "Age Of Empires 3",
+    "description": "Full list of all Age of Empires 3 cheat codes",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/age-of-empires-3.json]: " List of Age of Empire 3 cheat codes "
Would be enough here

**Comment by zekiel** [comment]
OK to merge! Thanks @yasharmaster! This and #3256 will be great! 

as @jophab mentioned, the description could be a bit shorter but I think 'full list' does add value so ok for now.

**Pull request merged**